recipes/leptonica/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "leptonica" %}
+{% set version = "1.76.0" %}
+{% set sha256 = "ff98ffef6103cda0e400676f81113a8157569ac0f453c6e03a1a27b305ba25a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DanBloomberg/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches: c74e712799890a600814d4d4970f8f03be254b0a.patch # fixes openjpeg
```

Comment:
Can you send a PR upstream and add a comment with the link?

Reply:
Will do.

Reply:
https://github.com/DanBloomberg/leptonica/pull/345